features/evidence_hub_search.feature

```diff
@@ -62,15 +62,49 @@ Feature: Evidence Hub Search
       | year of publication | 2015                                                     |
 
   Scenario: Clearing the filters for a filtered search
-    Given I visit the evidence hub search page
-    And I search based on some filters
+    Given I search based on some filters
     When I want to clear the filters
     Then I should see no filters checked
 
   Scenario: Clearing the filters for a filtered keyword search
-    Given I visit the evidence hub search page
-    And I search the evidence hub with a keyword "mortgage"
+    Given I search the evidence hub with a keyword "mortgage"
     And I search based on some filters
     When I want to clear the filters
     Then I should see no filters checked
     And I should see the keyword field cleared
+
+  Scenario: First page sending per page
+    Given I am on the page "1" seeing "1" evidence summary per page
```

Comment:
I *think* with the new cucumber syntax you no longer need to wrap arguments with `""`. I'm trying to find the documentation on this, but can't immediately find it, so I may be wrong with this